pom.xml

```diff
@@ -696,7 +696,6 @@
           </systemProperties>
           <reuseForks>true</reuseForks>
           <forkCount>${concurrency}</forkCount>
-          <argLine>-Djava.awt.headless=true ${argLine}</argLine>
```

Comment:
> I think this is a regression,

Regression happened when i switched to plugin-pom because it screwed plugin executions. If you want specify argline or headless, you still have this option.